js-miniapp-sample/package.json

```diff
@@ -7,6 +7,7 @@
     "@material-ui/icons": "^4.9.1",
     "axios": "^0.19.2",
     "bowser": "^2.9.0",
+    "js-miniapp-sdk": "^1.1.0",
```

Comment:
I see js-miniapp-sdk on npm has versions only [ '0.1.0-alpha', '1.0.0', '1.1.0-alpha', '1.1.0' ] but don't know why JIRA description has 1.2.0.
![image](https://user-images.githubusercontent.com/61268229/89624177-d36aa400-d8d0-11ea-8037-497f403f470b.png)
https://jira.rakuten-it.com/jira/browse/MINIBUILD-344
@corycaywood  is 1.2.0 version exist?

Reply:
I dont think they have it published
But the latest sample sdk app has that inbuilt
![Screen Shot 2020-08-07 at 17 19 53](https://user-images.githubusercontent.com/67311553/89625185-5f310000-d8d2-11ea-9dc9-fa3d790c8df5.png)

Reply:
please ignore it. updated the JIRA issue description. Ashton confirmed the latest sdk version is 1.1.0

Reply:
The iOS/Android SDKs and the JS SDK are published independently of each other because we sometimes need to release updates for the native code but not the JS code. So the iOS/Android SDKs were on 1.2.0 (now the latest is 2.0.0), but the latest version of the JS SDK is 1.1.0. See the [changelog](https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-sdk/CHANGELOG.md).